.github/workflows/release_pull_request.yml

```diff
@@ -15,14 +16,13 @@ on:
       devPrUrl:
         value: ${{jobs.create_prs.outputs.devPrUrl}}
       releaseHash:
-        value: ${{jobs.create_prs.outputs.releaseHash}}
+        value: ${{jobs.create_prs.outputs.releaseHash}}     
   workflow_dispatch:
     inputs:
       version:
         description: 'Version Number (eg. v1.1.0)'
         required: true
         type: string
-
```

Comment:
Would maybe add this line break back in to have separation between `on:` and `jobs:`